src/Documentation/Getting-Started-With-Orleans/Developing-a-Client.md

```diff
@@ -3,71 +3,87 @@ layout: page
 title: Developing a Client
 ---
 
+### What Is Grain Client?
 
-Once we have our grain type implemented, we can write a client application that uses the type.
+The term "Client" or sometimes "Grain Client" is used for application code that interacts with grains but itself is not part of a grain logic.
+Client code runs outside of the cluster of [Orleans servers called "silos"](Silos.md) where grains are hosted.
+Hence, a client acts as a connector or conduit to the cluster and to all grains of the application.
```

Comment:
For future docs we should try to use simple, casual sentences to make the docs more accessible to readers who are less fluent in English. I need to keep reminding myself